app/src/main/java/org/p2p/wallet/debug/settings/DebugSettingsAdapter.kt

```diff
@@ -56,6 +56,7 @@ class DebugSettingsAdapter(
             is SettingsRow.Info -> R.layout.item_settings_info
             is SettingsRow.Logout -> R.layout.item_settings_logout
             is SettingsRow.Switcher -> R.layout.item_settings_debug_switch
+            else -> error("")
```

Comment:
give some good message, even for debug purposes

Reply:
yeah sure, im gonna clean this all up